test/db/analysis/vars

```diff
@@ -870,58 +873,233 @@ pdf
+.(sig)
+s dbg.prout
+.(sig)
+s dbg._fpadd_parts
+.(sig)
+s dbg.strnlen_s
+.(sig)
+s dbg._Wctomb
+.(sig)
+s dbg._Mbtowc
+.(sig)
+EOF
+EXPECT=<<EOF
+var va_list ap @ stack + 0x4
+arg const char * fmt @ a6
+int printf(const char *format);
+            ; CALL XREFS from dbg.main @ 0x8000054e, 0x80000636
+            ;-- printf:
+/ dbg.printf(const char * fmt,va_args ...);
+|           ; arg const char *fmt @ a6
+|           ; var va_list ap @ stack + 0x4
+|           0x80000c22      mov.aa %a6, %a4                            ; printf.c:10 ; arg5 ; int printf(const char * fmt,va_args ...);
+
+arg void * str @ a15
+arg const char * buf @ a5
+arg size_t n @ d4
+void dbg.prout(const char *buf);
+            ;-- prout:
+/ * dbg.prout(void * str,const char * buf,size_t n);
+|           ; arg void *str @ a15
+|           ; arg size_t n @ d4
+|           ; arg const char *buf @ a5
+|           0x80000c04      mov.aa %a15, %a4                           ; printf.c:5 ; arg5 ; void * prout(void * str,const char * buf,size_t n);
+
+arg fp_number_type * b @ a12
+arg fp_number_type * a @ a13
+arg fp_number_type * tmp @ a15
+arg fractype a_fraction @ d11
+arg int diff @ d12
+arg int a_normal_exp @ d15
+arg int b_normal_exp @ d2
+arg int sdiff @ d3
+arg fractype b_fraction @ d9
+void dbg._fpadd_parts();
+            ; CALL XREF from dbg.__adddf3 @ 0x80003e32
+            ; CALL XREF from dbg.__subdf3 @ 0x80003e72
+            ;-- _fpadd_parts:
+/ * dbg._fpadd_parts(fp_number_type * a,fp_number_type * b,fp_number_type * tmp);
+|           ; arg fp_number_type *a @ a13
+|           ; arg fp_number_type *b @ a12
+|           ; arg fp_number_type *tmp @ a15
+|           ; arg int sdiff @ d3
+|           ; arg int a_normal_exp @ d15
+|           ; arg int b_normal_exp @ d2
+|           ; arg fractype a_fraction @ d11
+|           ; arg fractype b_fraction @ d9
+|           ; arg int diff @ d12
+|           0x80003c60      ld.bu %d15, [%a4]0                         ; fp-bit.c:604 ; arg5 ; fp_number_type * _fpadd_parts(fp_number_type * a,fp_number_type * b,fp_number_type * tmp);
+
+arg const char * sc @ a2
+arg const char * s @ a4
+arg size_t maxsize @ d4
+void dbg.strnlen_s(const char *s);
+            ; CALL XREF from dbg._Fail_s @ 0x8000191e
+            ;-- strnlen_s:
+/ dbg.strnlen_s(const char * s,size_t maxsize);
+|           ; arg size_t maxsize @ d4
+|           ; arg const char *sc @ a2
+|           ; arg const char *s @ a4
+|           0x800030ca      mov   %d2, 0                               ; strnlen_s.c:6 ; size_t strnlen_s(const char * s,size_t maxsize);
+
+arg mbstate_t * pst @ a12
+arg _Statab * pwcstate @ a13
+arg char * s @ a15
+arg wchar_t wc @ d15
+void dbg._Wctomb();
+            ; CALL XREF from dbg._Putstr @ 0x800014d2
+            ;-- _Wctomb:
+/ dbg._Wctomb(char * s,wchar_t wc,mbstate_t * pst);
+|           ; arg char *s @ a15
+|           ; arg wchar_t wc @ d15
+|           ; arg mbstate_t *pst @ a12
+|           ; arg _Statab *pwcstate @ a13
```

Comment:
In this test and others please use `var` for any variable that is non-argument one.